### ISSUE TITLE
Idea: Sharing productivity score globally

### ISSUE THREAD

**Issue by ideamonk** [gurgeh/selfspy#23]
This is great! Once #22 is done and this also works over OS X, something nice to have - 
1. Let users score entities (URLs, applications) on productivity, say -3 to 3
2. Have a global score base constantly updated by many users
3. Use this score data to automatically score new users

This is very similar to RescueTime, something we'd <3 to have. Keeping the score db open, unlike RescueTime = more win for all.

Sidenote: Words < Action, I know. I wish to, but don't have time to contribute due to certain personal goals this year.
